Markdown/syntax_test_multimarkdown.md

```diff
@@ -29,7 +29,8 @@ HTML Header: <style>
              </style>
 T:           ^^^^^^^^ meta.header.multimarkdown string.unquoted.multimarkdown
 
-T: <- meta.content.multimarkdown - meta.header.multimarkdown
+T: 
```

Comment:
FYI, there's a trailing space here.

Reply:
Good catch, while the bigger issue is `T:` to be the correct test case indicator - not `|` as assumed in the last tests.